src/index.js

```diff
@@ -51,39 +50,23 @@ module.exports = {
   Transloadit,
   AwsS3,
   Dashboard,
-  MetaData,
   Webcam,
   GoldenRetriever,
   ReduxDevTools,
   ReduxStore
 }
 
-Object.defineProperty(module.exports, 'RestoreFiles', {
-  enumerable: true,
-  configurable: true,
-  get: () => {
-    console.warn('Uppy.RestoreFiles is deprecated and will be removed in v0.22. Use Uppy.GoldenRetriever instead.')
-    Object.defineProperty(module.exports, 'RestoreFiles', {
-      enumerable: true,
-      configurable: true,
-      writable: true,
-      value: GoldenRetriever
-    })
-    return GoldenRetriever
-  }
-})
-
-Object.defineProperty(module.exports, 'Tus10', {
-  enumerable: true,
-  configurable: true,
-  get: () => {
-    console.warn('Uppy.Tus10 is deprecated and will be removed in v0.22. Use Uppy.Tus instead.')
-    Object.defineProperty(module.exports, 'Tus10', {
-      enumerable: true,
-      configurable: true,
-      writable: true,
-      value: Tus
-    })
-    return Tus
-  }
-})
+// Object.defineProperty(module.exports, 'Tus10', {
```

Comment:
lets get rid of this commented bit too

Reply:
How will we remember it next time we need to rename a plugin? :)

Reply:
by checking the git history! :D

Reply:
You’re such a millennial!